### DISCUSSION TITLE
How to change default text color

### DISCUSSION THREAD

**Discussion by PazminoJose** [KevinVandy/mantine-react-table#102] [Q&A]
I have been trying to change the text color  of the table head cell but the default color takes priority over the color i set, i tried with the sx property and with tailwind but it doesn´t work, I´ll like to change it because  I set a green background for the table head and a white label looks better.
I tried to set the color like this
```js
  mantineTableHeadRowProps: {
            className: "bg-green"
        },
        mantineTableHeadCellProps: {
            className: "text-white",
            sx: {
                color: "white"
            }
        },
```
But the result its the same

![Untitled](https://github.com/KevinVandy/mantine-react-table/assets/84350058/683b004f-3b7d-432b-87b8-8ebee07d0a91)

this happen with the filter input too

**Comment by KevinVandy** [comment]
Usually you have to target specific CSS classes within the `sx` or `styles` prop if customizing Mantine Components

https://mantine.dev/styles/styles-api/

You can also change styles and colors globaly in the Mantine Theme provider.
